static/app/types/index.tsx

```diff
@@ -1978,11 +1978,22 @@ export type Frame = {
   symbolicatorStatus: SymbolicatorStatus;
   trust: any | null;
   vars: Record<string, any> | null;
+  addrMode?: string;
   origAbsPath?: string | null;
   mapUrl?: string | null;
   map?: string | null;
+  isSentinel?: boolean;
+  isPrefix?: boolean;
+  minGroupingLevel?: number;
 };
 
+export enum FrameCategory {
+  SENTINEL = 'sentinel',
+  PREFIX = 'prefix',
+  GROUPING = 'grouping',
+  IN_APP = 'in_app',
+}
```

Comment:
a frame can have all of those at once

also category is already a used term in the backend, would not recommend using that term here